music_album.rb

```diff
@@ -0,0 +1,36 @@
+require 'securerandom'
+require 'date'
+require './item'
+
+class MusicAlbum < Item
+  attr_accessor :genre, :on_spotify
+
+  def initialize(publish_date, on_spotify, archived, id = SecureRandom.uuid)
+    super(publish_date, archived, id)
+    @on_spotify = on_spotify
+  end
+
+  def as_obj
+    {
+      id: @id,
+      publish_date: {
+        year: @publish_date.year,
+        month: @publish_date.month,
+        day: @publish_date.day
+      },
+      on_spotify: @on_spotify,
+      archived: @archived,
+      label_id: @label&.id || ''
+    }
+  end
+
+  def to_json(*_args)
+    as_obj.to_json
+  end
+
+  private
+
+  def can_be_archived?
+    super && @on_spotify
```

Comment:
- [ ] According to the project requirement you should return true if parent's method returns true AND if on_spotify equals true, you check the example below ⬇️ 
```ruby
 def can_be_archived?
      super && @on_spotify == true
    end
```

Reply:
@codecaiine `super && @on_spotify` and `super && @on_spotify == true` have the exact same behavior. Actually, the solution you propose is more verbose. Think about it, if `@on_spotify` is true and `super` returns true, then the `&` operator will return `true` otherwise if one of those values is `false` It will also return false, so there's no need to verify if `@on_spotify` is true, since if it is true, it will return true, the exact same behavior can be achieved just by writing `@on_spotify`

Reply:
Also, we added tests to demonstrate that the code works.

![image](https://user-images.githubusercontent.com/26579035/234922141-24fbb313-c5a5-4390-a6f1-e564434cbb8b.png)